city_scrapers/spiders/chi_ssa_27.py

```diff
@@ -0,0 +1,137 @@
+from datetime import datetime
+from re import split, sub
+
+from city_scrapers_core.constants import COMMISSION
+from city_scrapers_core.items import Meeting
+from city_scrapers_core.spiders import CityScrapersSpider
+from scrapy import Field, Item
+
+
+class ChiSsa27Spider(CityScrapersSpider):
+    name = "chi_ssa_27"
+    agency = "Chicago Special Service Area #27 Lakeview West"
+    timezone = "America/Chicago"
+    allowed_domains = ["lakeviewchamber.com"]
+    start_urls = ["https://www.lakeviewchamber.com/ssa27"]
+    minutes_list = []
+
+    def parse(self, response):
+        """   `parse` should always `yield` Meeting items. """
+        self.minutes_list = self.get_minutes_panel_items(response)
+        self._validate_locations(response)
+        commission_path = "div #content-232764 div.panel-body p"
+        for item in response.css(commission_path)[1:]:  # main
+            meeting = Meeting(
+                title=self._parse_title(item),
+                description='',
+                classification='',
```

Comment:
Guessing you just haven't gotten to this yet, but this would be the `COMMISSION` constant

Reply:
I got confused with Committees in the mix. Will change to just COMMISSION.